Proposing WikiSplit metric card (#4098)
huggingface/datasets: metrics/wiki_split/README.md

> # Metric Card for WIKI_SPLIT

Reviewer: ```suggestion
# Metric Card for WikiSplit
```

## Metric description

> WIKI_SPLIT is the combination of three metrics: [SARI](https://huggingface.co/metrics/sari), [EXACT_MATCH](https://huggingface.co/metrics/exact_match) and [SACREBLEU](https://huggingface.co/metrics/sacrebleu). 

Reviewer: ```suggestion
WikiSplit is the combination of three metrics: [SARI](https://huggingface.co/metrics/sari), [exact match](https://huggingface.co/metrics/exact_match) and [SacreBLEU](https://huggingface.co/metrics/sacrebleu). 
```

## How to use 

The WIKI_SPLIT metric takes three inputs:

`sources`: a list of source sentences, where each sentence should be a string.

`predictions`: a list of predicted sentences, where each sentence should be a string.

`references`: a list of lists of reference sentences, where each sentence should be a string.

```python
>>> from datasets import load_metric
>>> wiki_split = load_metric("wiki_split")
>>> sources = ["About 95 species are currently accepted ."]
>>> predictions = ["About 95 you now get in ."]
>>> references= [["About 95 species are currently known ."]]
>>> results = wiki_split.compute(sources=sources, predictions=predictions, references=references)
```
## Output values

This metric outputs a dictionary containing three scores:

`sari`: the [SARI](https://huggingface.co/metrics/sari) score, whose range is between `0.0` and `100.0` -- the higher the value, the better the performance of the model being evaluated, with a SARI of 100 being a perfect score.

> `sacrebleu`: the [SACREBLEU](https://huggingface.co/metrics/sacrebleu) score, which can take any value between `0.0` and `100.0`, inclusive.

Reviewer: ```suggestion
`sacrebleu`: the [SacreBLEU](https://huggingface.co/metrics/sacrebleu) score, which can take any value between `0.0` and `100.0`, inclusive.
```

> `exact`: the [EXACT_MATCH](https://huggingface.co/metrics/exact_match) score, which represents the sum of all of the individual exact match scores in the set, divided by the total number of predictions in the set. It ranges from `0.0` to `100`, inclusive. Here, `0.0` means no prediction/reference pairs were matches, while `100.0` means they all were.

Reviewer: ```suggestion
`exact`: the [exact match](https://huggingface.co/metrics/exact_match) score, which represents the sum of all of the individual exact match scores in the set, divided by the total number of predictions in the set. It ranges from `0.0` to `100`, inclusive. Here, `0.0` means no prediction/reference pairs were matches, while `100.0` means they all were.
```
